Automattic/wp-calypso
# Themes: Remove Flux stores

To test:

* Make sure the Themes showcase (`calypso.localhost:3000/design`) works as before.
* Run `make test` inside `shared/lib/themes`

Changes:

* `reducers/themes-last-query`: Remove Flux dispatcher `waitFor`
* `reducers/themes-list`: Use `isJetpack()` from selectors instead of
  `ThemeLastQueryStore`'s
* Kill `shared/lib/themes/stores`
* Remove obsolete helper functions from reducers, now that they are in `selectors/`
* Default-export reducers, don't export initialStates anymore
* Update README.md
* Rewrite tests in `shared/lib/themes` to directly test reducers instead of stores
  Note that in test/themes-list-store, the Jetpack test is removed since filtering
	is no longer done in the reducer but in the themes-list-fetcher data component
  (using the getFilteredThemes()) now.
